pkg/datasource/csvsource/testdata/BTCUSDT-1h-2021-Q1.csv

```diff
@@ -0,0 +1,2158 @@
+1609459200000,28923.63000000,29031.34000000,28690.17000000,28995.13000000,2311.81144500,1609462799999,66768830.34010008,58389,1215.35923800,35103542.78288276,0
+1609462800000,28995.13000000,29470.00000000,28960.35000000,29409.99000000,5403.06847100,1609466399999,158357816.81805722,103896,3160.04170100,92613991.93555293,0
+1609466400000,29410.00000000,29465.26000000,29120.03000000,29194.65000000,2384.23156000,1609469999999,69842653.67342030,57646,1203.43350600,35252749.90832606,0
+1609470000000,29195.25000000,29367.00000000,29150.02000000,29278.40000000,1461.34507700,1609473599999,42760776.72551646,42510,775.91566600,22705547.98307977,0
+1609473600000,29278.41000000,29395.00000000,29029.40000000,29220.31000000,2038.04680300,1609477199999,59614637.30352874,55414,1003.34283400,29346381.88020654,0
+1609477200000,29220.31000000,29235.28000000,29084.11000000,29187.01000000,1469.95626200,1609480799999,42864538.70435811,41800,679.84674200,19827190.29247262,0
+1609480800000,29187.01000000,29270.00000000,29077.32000000,29174.35000000,1420.72629100,1609484399999,41446013.01819005,46400,699.14267600,20398322.75992173,0
+1609484400000,29174.35000000,29191.98000000,28806.54000000,29092.83000000,2380.18091800,1609487999999,69034619.09489940,53158,1054.72099100,30598457.40248306,0
+1609488000000,29092.84000000,29178.03000000,28872.24000000,29000.01000000,2008.16573900,1609491599999,58274190.69862189,55012,1022.06617000,29662447.72598589,0
+1609491600000,29000.01000000,29307.73000000,28970.00000000,29202.21000000,2022.05602200,1609495199999,59006512.26098600,43674,1208.47757800,35272717.35224266,0
+1609495200000,29202.21000000,29344.97000000,29152.88000000,29223.82000000,1944.25584100,1609498799999,56881923.87605724,46783,1014.53831900,29683027.81899852,0
+1609498800000,29223.82000000,29402.57000000,29212.44000000,29313.49000000,2309.22771700,1609502399999,67680909.41227815,49883,1309.01281600,38371497.60008983,0
+1609502400000,29313.49000000,29600.00000000,29150.00000000,29233.49000000,4191.91516100,1609505999999,123253838.23617110,82845,2333.30675300,68649197.19260735,0
```

Comment:
this file is too big for just testing, could you simply keep 100~200 lines since it's only used for testing? thanks

Reply:
yes gonna do that